### ISSUE TITLE
how to create a s3 sink connector that will dump a stream/table to s3 

### ISSUE THREAD

**Issue by jma562** [confluentinc/ksql#9641]
Hi,

I'm working on building a kafka connector  to dump kafka messages to s3 storage in parquet format.
I have an option to leverage ksqldb to massage the kafka record to remove empty struct which cannot be handled 
by paquet and will cause an exception in connector to transform avro->parquet.

The question is, once after the data is processed by ksql, how can I use kafka connector to dump data to s3.
Can kafka connector be used as a sink connector that consumes from ksqldb directly?
Or I have to move the stream/table back to kafka under a new topic?

Hopefully there's a connector that can be directly invoked by ksqldb.  Thanks.

**Comment by pgaref** [comment]
Hey @jma562  have you checked?
https://docs.confluent.io/kafka-connectors/s3-sink/current/overview.html#amazon-s3-sink-connector-for-cp

**Comment by jma562** [comment]
Hi @pgaref 
I was able to set up s3 sink connector to dump kafka topic messages to s3.
However, I'm not quite clear after I leverage ksql db to process data, how can I dump the data after being processed by ksql db to s3. I've read that you can create a sink connector in ksql, but how to configure the s3 sink connector to consume data from ksql (instead of the original data in kafka topic). I have not seen any clear document/examples yet.  Thanks.

**Comment by danielloader** [comment]
I agree that this is a good use-case to document, taking snapshots of a stream that's filtered by sql queries and storing them in object store is a unique selling point here and would love to have a good well documented example to work from.
